vizro-ai/docs/index.md

```diff
@@ -64,11 +49,19 @@ Vizro-AI leverages the power of [Plotly](https://plotly.com/python/) to produce
     <div class="card-content">
       <h5>Contribute</h5>
       <p>
-        Find our contribution guidelines and all our current and previous contributors here.
+        Contribution guidelines and all our current and past contributors.
       </p>
     </div>
   </div>
 </a>
 
 </div>
 </div>
+
+!!! notice "Notice"
+    Please review this [disclaimer](pages/explanation/disclaimer.md)
+    before using the `vizro-ai` package.
+
+    Users must connect to Large Language Models (LLMs) to use Vizro-AI.
+    Please review our [guidelines on the use of LLMs](pages/explanation/safety-in-vizro-ai.md)
+    and the required [safeguarding for dynamic code evaluation](pages/explanation/safeguard.md).
```

Comment:
Not sure what our common syntax is but here we capitalise `Large Language Models` while in other sections we have everything in lower-case. So whatever you think is the better choice, I would stick with that 👍 

```suggestion
    Users must connect to large language models (LLMs) to use Vizro-AI.
    Please review our [guidelines on the use of LLMs](pages/explanation/safety-in-vizro-ai.md)
    and the required [safeguarding for dynamic code evaluation](pages/explanation/safeguard.md).
```

Reply:
Thanks for the spot! I thought I'd caught all of those but I missed this one. Heading off to grep again. The rule is:

1. Capitalise in the acronym (so it's "LLM")
2. Don't capitalise the full name because it's not a proper noun (it's "large language models") UNLESS at the start of a sentence or header, then it's "Large language models"

Reply:
Perfect, all clear now! 👍